cdhweb/static_src/global/components/event-hero.scss

```diff
@@ -88,6 +90,12 @@ Event hero. Similar to standard hero. Key differences:
   }
 }
 
+.event-hero__type-tag {
+  justify-self: start;
+  align-self: start;
+  order: -1; // visually before h1
```

Comment:
@sarahframe this shouldn't stop you from merging this, but I can't pass up a good opportunity to nerd-snipe @liamjohnston and learn something new myself: would `reading-flow: flex-visual` be useful here? 
https://developer.chrome.com/blog/reading-flow

It feels like it would help from a **keyboard navigation** perspective (if the tag is a focusable element that takes you to a filtered list of articles with that tag, I dunno if it is, it might just be a `<span>`) 
But I don't really understand how `reading-flow` affects screen reader output. If (as I assume) it alters the screen reader output to read the tag first, then would be we undoing the decision to _nest_ the tag under the `h1` semantically?